x-pack/plugins/cases/public/components/case_view/components/assign_users.tsx

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useEffect, useState } from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiLink,
+  EuiLoadingSpinner,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { UserProfileWithAvatar } from '@kbn/user-profile-components';
+import { useAssignees } from '../../../containers/user_profiles/use_assignees';
+import { CaseAssignees } from '../../../../common/api/cases/assignee';
+import * as i18n from '../translations';
+import { SidebarTitle } from './sidebar_title';
+import { UserRepresentation } from '../../user_profiles/user_representation';
+import { useCasesContext } from '../../cases_context/use_cases_context';
+import { Assignee } from '../../user_profiles/types';
+import { SuggestUsersPopover } from './suggest_users_popover';
+import { CurrentUserProfile } from '../../types';
+
+interface AssigneesListProps {
+  assignees: Assignee[];
+  currentUserProfile: CurrentUserProfile;
+  assignSelf: () => void;
+  togglePopOver: () => void;
+  onAssigneeRemoved: (removedAssigneeUID: string) => void;
+}
+
+const AssigneesList: React.FC<AssigneesListProps> = ({
+  assignees,
+  currentUserProfile,
+  assignSelf,
+  togglePopOver,
+  onAssigneeRemoved,
+}) => {
+  return (
+    <>
+      {assignees.length === 0 ? (
+        <EuiFlexGroup direction="column" gutterSize="none">
+          <EuiFlexItem grow={false}>
+            <EuiText size="s" color="subdued">
+              <p>
+                {i18n.NO_ASSIGNEES}
+                <br />
+                <EuiLink data-test-subj="case-view-assign-users-link" onClick={togglePopOver}>
```

Comment:
For a user with `Read` access to Cases in Security, these actions are still enabled:

<img width="458" alt="image" src="https://user-images.githubusercontent.com/7405507/189158862-3f0ebd0e-a9eb-4afd-bd3f-ce5347c69183.png">

Reply:
Confirmed fixed by https://github.com/elastic/kibana/pull/140208/commits/d85963e5dd2b3c2a208cb67bbd9de5bb929a043e